backend/fms_core/tests/test_template_importers/project_link_samples_test.py

```diff
@@ -0,0 +1,64 @@
+from django.test import TestCase
+from django.contrib.contenttypes.models import ContentType
+import datetime
+
+from fms_core.template_importer.importers import ProjectLinkSamples
+from fms_core.tests.test_template_importers._utils import load_template, APP_DATA_ROOT
+
+from fms_core.models import SampleKind, SampleByProject
+
+from fms_core.services.container import create_container
+from fms_core.services.individual import get_or_create_individual
+from fms_core.services.sample import create_sample
+from fms_core.services.project import create_project
+
+
+
+class ProjectLinkSamplesTestCase(TestCase):
```

Comment:
A suggestion: for these tests, it would be even better to test the REMOVE action too in case :)